README.md

```diff
@@ -89,9 +89,12 @@ console.log('curl -i http://localhost:3000/users -d "name=test"');
 
 - `patchNode` **{Boolean}** Patch request body to Node's `ctx.req`, default `false`
 - `patchKoa` **{Boolean}** Patch request body to Koa's `ctx.request`, default `true`
-- `jsonLimit` **{String|Integer}** The byte (if integer) limit of the JSON body, default `1mb`
-- `formLimit` **{String|Integer}** The byte (if integer) limit of the form body, default `56kb`
-- `textLimit` **{String|Integer}** The byte (if integer) limit of the text body, default `56kb`
+- `jsonLimit` **{String|Integer|Function}** The byte (if integer) limit of the JSON body, default `1mb`
+  - If this is a `function`, it should take `ctx` as an argument.
```

Comment:
it's not _"it should take"_, it actually _takes_ it. So, "it takes `ctx` as (first) argument" sounds better.